$MirOS: ports/www/dillo/v2/patches/patch-dpid_Makefile_am,v 1.1 2008/11/07 18:05:52 bsiegert Exp $
--- dpid/Makefile.am.orig	Sun Jun  7 05:40:25 2009
+++ dpid/Makefile.am	Sun Aug 23 17:03:44 2009
@@ -1,4 +1,6 @@
 AM_CPPFLAGS=-DDPIDRC_SYS='"$(sysconfdir)/dpidrc"'
+sampledir = $(DESTDIR)$(datarootdir)/examples/dillo
+DPIDRC = $(sampledir)/dpidrc
 
 bin_PROGRAMS = dpid
 dpid_LDADD = ../dpip/libDpip.a ../dlib/libDlib.a
